firmware/Sources/HAL/HAL_STS/hal_sts.h

```diff
@@ -0,0 +1,86 @@
+/*********************************************************************************
+* @file           : hal_sts.h
+* @brief          : HAL header file for STS
+***********************************************************************************/
+
+#ifndef HAL_STS_H_
+#define HAL_STS_H_
+
+/**********************************************************************************/
+/*                               Project Includes                                 */
+/**********************************************************************************/
+#include "i2c.h"
```

Comment:
dont include this in .h. This will export this module to all modules wich import HAL_STS

Reply:
Changed in last commit